fluent-plugin-elasticsearch.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = Gem::Requirement.new(">= 2.0".freeze)
 
-  s.add_runtime_dependency 'fluentd', '>= 0.10.43'
+  s.add_runtime_dependency 'fluentd', '>= 0.14.0'
```

Comment:
Fluentd 0.14.4, 0.14.5 have various bugs related to its built-in plugins.
Could you specify 0.14.8 or later here?
0.14.8 seems to be relatively stable.